app/controllers/teachers/questions_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Teachers::QuestionsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :check_role
+
+  def index
+    @questions = Question.all
+  end
+
+  def show
+    @question = Question.find(params[:id])
+  end
+  def new
+	  @question = Question.new
```

Comment:
remove unnecessary indentations, remove commented code or leave comment why you need it, like 
`TODO: uncomment this when..`

Reply:
remove unnecessary indentation here, should be 2 spaces or 1 tab.